src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -643,14 +626,40 @@ def resume_with_manual_input(
             detail=f"Cannot resume privacy request '{privacy_request.id}'; no paused collection.",
         )
 
-    validate_manual_input(manual_rows, paused_collection, db)
-    logger.info(
-        f"Caching manual input for privacy request '{privacy_request_id}', collection: '{paused_collection}'"
-    )
-    privacy_request.cache_manual_input(paused_collection, manual_rows)
+    if paused_step != expected_paused_step:
```

Comment:
yep, I always cache them together at the same time, they either both exist (paused_collection and paused_step), or none exist, but I'll add an extra check just in case, maybe instead at `if not paused_collection` above.